bitnami/kafka/values.yaml

```diff
@@ -1698,13 +1701,43 @@ provisioning:
   ##
   waitForKafka: true
 
+## @section Kraft chart parameters
+
+## Kraft configuration
+## Kafka mode without Zookeeper. Kafka nodes can work as controllers in this mode.
+##
+kraft:
+  ## @param kraft.enabled Switch to enable or disable the Kraft mode for Kafka
+  ##
+  enabled: false
+  ## @param kraft.controllerPort Kafka Controller listener port
+  ##
+  controllerPort: 9095
```

Comment:
Added the ports for headless-svc and for the kafka-container. I intentionally did not define .Values.service.ports.controller because we need only headless service for the controller. And for the headless service controller port will be always equal to service port